### PR TITLE
docs: document server class "all"

### PR DESCRIPTION
**Pull request by AlekSi**
Refs #310.

### CODE REVIEW

File: docs/website/content/docs/v0.3/Configuration/serverclasses.md [hunk 1]
```diff
@@ -31,3 +31,5 @@ spec:
 ```
 
 Servers would only be added to the above class if they had _EITHER_ CPU info, _AND_ the label associated with the server resource.
+
+Additionally, Sidero automatically creates and maintains a server class called `"all"` that includes all (accepted) servers.
```
Comment (AlekSi) [inline docs/website/content/docs/v0.3/Configuration/serverclasses.md:35]: Should it be "all" or "any"?

Comment (Ulexus) [reply docs/website/content/docs/v0.3/Configuration/serverclasses.md:35]: Any sounds like a better name to me, yes.

File: docs/website/content/docs/v0.3/Guides/first-cluster.md [hunk 2]
```diff
@@ -128,9 +127,9 @@ Note that there are several variables that should be set in order for the templa
 - `CONTROL_PLANE_SERVERCLASS`: The server class to use for control plane nodes.
 - `WORKER_SERVERCLASS`: The server class to use for worker nodes.
 - `KUBERNETES_VERSION`: The version of Kubernetes to deploy (e.g. `v1.19.4`).
-- `TALOS_VERSION`: This should correspond to the minor version of Talos that you will be deploying (e.g. `v0.9`).
+- `TALOS_VERSION`: This should correspond to the minor version of Talos that you will be deploying (e.g. `v0.10`).
   This value is used in determining the fields present in the machine configuration that gets generated for Talos nodes.
-  Note that the default is currently `v0.8`.
```
Comment (smira) [inline docs/website/content/docs/v0.3/Guides/first-cluster.md:133]: default is still `0.8` atm to maintain backwards compatibility (it's in the CABPT)

**Comment by AlekSi** [comment]
https://github.com/talos-systems/sidero/pull/373

**Pull request closed without merging**